### PR TITLE
virsh_node_memtune: Add cases for options combination test

### PR DESCRIPTION
**Pull request by hahakiki2010**
1) Add cases for options combination test
2) Add case for virsh manual check

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
Test passed:
(virtenv_py3) [root@localhost tp-libvirt]# avocado run --vt-type libvirt virsh.node_memtune
JOB ID     : da43beb8891f4967f5af9d84aaead3b3d05b8509
JOB LOG    : /root/avocado/job-results/job-2018-07-18T11.48-da43beb/job.log
 (01/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.check_virsh_manual: PASS (3.35 s)
 (02/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.get_node_memory_parameter: PASS (3.25 s)
 (03/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_pages_to_scan.normal: PASS (3.27 s)
 (04/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_pages_to_scan.zero: PASS (3.57 s)
 (05/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_sleep_millisecs.normal: PASS (3.22 s)
 (06/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_sleep_millisecs.zero: PASS (3.22 s)
 (07/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_merge_across_nodes.disable: PASS (3.26 s)
 (08/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.shm_merge_across_nodes.enable: PASS (3.26 s)
 (09/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.positive_testing.set_node_memory_parameter.options_combination.shm_pages_to_scan.shm_sleep_millisecs: PASS (3.21 s)
 (10/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.get_node_memory_parameter: PASS (3.23 s)
 (11/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_value.shm_pages_to_scan: PASS (3.25 s)
 (12/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_value.shm_sleep_millisecs: PASS (3.25 s)
 (13/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_value.shm_merge_across_nodes.invalid_number: PASS (3.30 s)
 (14/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_value.shm_merge_across_nodes.invalid_operation: PASS (3.20 s)
 (15/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_options.option.shm_pages_to_scan: PASS (3.23 s)
 (16/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_options.option.shm_sleep_millisecs: PASS (3.39 s)
 (17/17) type_specific.io-github-autotest-libvirt.virsh.node_memtune.negative_testing.set_node_memory_parameter.invalid_options.option.shm_merge_across_nodes: PASS (3.24 s)
RESULTS    : PASS 17 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 178.94 s

**Comment by iccaszhulili** [comment]
rh-caselink retest please

File: libvirt/tests/cfg/virsh_cmd/host/virsh_node_memtune.cfg [hunk 1]
```diff
@@ -30,6 +32,13 @@
                                     shm_merge_across_nodes = 0
                                 - enable:
                                     shm_merge_across_nodes = 1
+                        - options_combination:
```
Comment (balamuruhans) [inline libvirt/tests/cfg/virsh_cmd/host/virsh_node_memtune.cfg]: this is already taken care in the existing test variants, `shm_pages_to_scan:` and `shm_sleep_millisecs:` multiplex to form the combination.

File: libvirt/tests/src/virsh_cmd/host/virsh_node_memtune.py [hunk 2]
```diff
@@ -150,12 +151,24 @@ def set_node_memtune_parameter(test, params):
...
         else:
             if check_node_memtune(params, ksm_dicts):
-                logging.info(result)
+                logging.info(result.stdout)
             else:
                 test.fail("The memory parameters "
                           "mismatch with result")
 
 
+def check_virsh_manual(cmd, test):
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/host/virsh_node_memtune.py]: why do we have virsh manual test in virsh_node_memtune.py ?

**Pull request merged**